src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/config.jelly

```diff
@@ -10,15 +10,19 @@
        <f:textbox />
     </f:entry>
 
-    <f:entry field="hubAddress" title="Hub address">
+    <f:entry field="hubAddress" title="Hub Address">
         <f:textbox/>
     </f:entry>
     
-    <f:entry field="projectName" title="Project name">
+    <f:entry field="projectName" title="CodeSonar hub project path">
```

Comment:
This is the field title seen in the Freestyle project GUI.  

I don't think we have consensus on a better name for this field yet, so I think we should not change the field name in this version.  The title was "Project name"; I think (at least for now), we should keep that title.  Maybe change casing to "Project Name" to make it match the other field titles that have been modified here.


"CodeSonar hub project path" was my suggestion in another thread, but I don't think we all agree that it is the best title.